### PR TITLE
Adding the latest integrations and community channels

### PR DESCRIPTION
**Pull request by vinijaiswal**
- Added a separate section for Delta Stand-alone.
- Added the connectors information about Apache Hive, Apache Flink, PrestoDB, Trino, Rust and Kafka-delta-ingest
- Added Delta Lake LinkedIn and youtube channels.

### CODE REVIEW

**Comment by vinijaiswal** [comment]
- Added a separate section for Delta Stand-alone.
- Added the connectors information about Apache Hive, Apache Flink, PrestoDB, Trino, Rust and Kafka-delta-ingest
- Added Delta Lake LinkedIn and youtube channels.

File: README.md [hunk 1]
```diff
@@ -1,14 +1,31 @@
...
-Delta Lake is a storage layer that brings scalable, ACID transactions to [Apache Spark](https://spark.apache.org) and other big-data engines.
-
-See the [Delta Lake Documentation](https://docs.delta.io) for details.
-
-See the [Quick Start Guide](https://docs.delta.io/latest/quick-start.html) to get started with Scala, Java and Python.
+Delta Lake is an open-source storage framework that enables building a [Lakehouse architecture](http://cidrdb.org/cidr2021/papers/cidr2021_paper17.pdf) with compute engines including Spark, PrestoDB, Flink, Trino, and Hive and APIs for Scala, Java, Rust, Ruby, and Python. 
+* See the [Delta Lake Documentation](https://docs.delta.io) for details.
+* See the [Quick Start Guide](https://docs.delta.io/latest/quick-start.html) to get started with Scala, Java and Python.
+
+### Note, this repo is one of many Delta Lake repositories in the [delta.io](https://github.com/delta-io) organizations including
```
Comment (dennyglee) [inline README.md]: Please change the "Note, this repo" as the third bullet point after Delta Lake documentation and Quick Start Guide.

File: README.md [hunk 2]
```diff
@@ -1,14 +1,31 @@
...
+[delta-rs](https://github.com/delta-io/delta-rs),
+[delta-sharing](https://github.com/delta-io/delta-sharing),
+[kafka-delta-ingest](https://github.com/delta-io/kafka-delta-ingest), and
+[website](https://github.com/delta-io/website)
+
+## Integrations with Delta Lake:
+The following are some of the more popular Delta Lake integrations, refer to [delta.io/integrations](https://delta.io/integrations/) for the complete list.
+
+* [Apache Spark™](https://docs.delta.io/): This connector allows Apache Spark™ to read from and write to Delta Lake.
+* [Apache Flink (Preview)](https://github.com/delta-io/connectors/tree/master/flink-connector): This connector allows Apache Flink to write to Delta Lake.
```
Comment (dennyglee) [inline README.md:22]: Please change link to: https://github.com/delta-io/connectors/tree/master/flink

File: README.md [hunk 3]
```diff
@@ -21,17 +38,16 @@ See the [online documentation](https://docs.delta.io/latest/) for the latest rel
...
-See the [online documentation](https://docs.delta.io/latest/releases.html) for the releases and their compatibility with Apache Spark versions.
 
 ### API Compatibility
 
 There are two types of APIs provided by the Delta Lake project. 
 
-- Spark-based APIs - You can read Delta tables through the `DataFrameReader`/`Writer` (i.e. `spark.read`, `df.write`, `spark.readStream` and `df.writeStream`). Options to these APIs will remain stable within a major release of Delta Lake (e.g., 1.x.x).
 - Direct Java/Scala/Python APIs - The classes and methods documented in the [API docs](https://docs.delta.io/latest/delta-apidoc.html) are considered as stable public APIs. All other classes, interfaces, methods that may be directly accessible in code are considered internal, and they are subject to change across releases.
+- Spark-based APIs - You can read Delta tables through the `DataFrameReader`/`Writer` (i.e. `spark.read`, `df.write`, `spark.readStream` and `df.writeStream`). Options to these APIs will remain stable within a major release of Delta Lake (e.g., 1.x.x).
+
```
Comment (dennyglee) [inline README.md:49]: Please add an indented bullet that has for `See the [online documentation](https://docs.delta.io/latest/releases.html) for the releases and their compatibility with Apache Spark versions.`

File: README.md [hunk 4]
```diff
@@ -41,7 +57,8 @@ Breaking changes in the protocol are indicated by incrementing the minimum reade
 
 ## Roadmap
```
Comment (dennyglee) [inline README.md:57]: Under the Data Storage Compatibility section, please update the protocol action link to `https://github.com/delta-io/delta/blob/master/core/src/test/scala/org/apache/spark/sql/delta/ActionSerializerSuite.scala`

**Review by dennyglee — CHANGES_REQUESTED** [review]
Looking great - please update per comments - thanks!

File: README.md (hunk 1 above)
Comment (vinijaiswal) [reply README.md]: I changed it in the a third file change. Can you please review all 3 changes together? Thank you!

File: README.md [hunk 5]
```diff
@@ -1,14 +1,52 @@
...
+* See the [Quick Start Guide](https://docs.delta.io/latest/quick-start.html) to get started with Scala, Java and Python.
+* Note, this repo is one of many Delta Lake repositories in the [delta.io](https://github.com/delta-io) organizations including 
+[connectors](https://github.com/delta-io/connectors),
+[delta](https://github.com/delta-io/delta), 
+[delta-rs](https://github.com/delta-io/delta-rs),
+[delta-sharing](https://github.com/delta-io/delta-sharing),
+[kafka-delta-ingest](https://github.com/delta-io/kafka-delta-ingest), and
+[website](https://github.com/delta-io/website).
+
+## Integrations with Delta Lake:
```
Comment (dennyglee) [inline README.md:18]: Please remove this header - thanks!

**Review by dennyglee — APPROVED** [review]

File: README.md [hunk 6]
```diff
@@ -21,27 +58,28 @@ See the [online documentation](https://docs.delta.io/latest/) for the latest rel
...
+- See the [online documentation](https://docs.delta.io/latest/releases.html) for the releases and their compatibility with Apache Spark versions.
+
 
 ### Data Storage Compatibility
 
 Delta Lake guarantees backward compatibility for all Delta Lake tables (i.e., newer versions of Delta Lake will always be able to read tables written by older versions of Delta Lake). However, we reserve the right to break forward compatibility as new features are introduced to the transaction protocol (i.e., an older version of Delta Lake may not be able to read a table produced by a newer version).
 
-Breaking changes in the protocol are indicated by incrementing the minimum reader/writer version in the `Protocol` [action](https://github.com/delta-io/delta/blob/master/src/main/scala/org/apache/spark/sql/delta/actions/actions.scala).
+Breaking changes in the protocol are indicated by incrementing the minimum reader/writer version in the `Protocol` [action](https://github.com/delta-io/delta/blob/master/core/src/test/scala/org/apache/spark/sql/delta/ActionSerializerSuite.scala).
```
Comment (zsxwing) [inline README.md:77]: why change this link?

**Pull request merged**